README.md

```diff
@@ -1,2 +1,28 @@
 # agent-packaging
 Combined packaging scripts for the Stackdriver monitoring agent
+
+## Branches in Use
+
+* master
```

Comment:
[Optional] Use code font for branch names, e.g.:
```markdown
* `master`
  ...
* `stackdriver-agent-5.5.2`
  ...
```
to distinguish them from, e.g., "Development branches" below.

Reply:
Done